docs/api/authentication.md

```diff
@@ -0,0 +1,72 @@
+# Authenticating and using the Agenda APIs
+
+This document provides a step-by-step guide on authenticating with the Agenda application so you can use the Agenda APIs. By following this guide, you will have generated the following credentials for your user account:
+
+- `{ACCESS_TOKEN}`
+- `{API_KEY}`
```

Comment:
Sounds good to me. I also just realized that we don't really generate the `{API_KEY}` field, since that's just the `{CLIENT_ID}` which will be automatically generated as part of the UI stuff. 

That being said, `{ACCESS_TOKEN}` seems to be pretty common vernacular, from my research on doing OAuth flows, so do you think we'll need to give a proper definition for it?

WRT Access Token being commonly used: 
- https://www.digitalocean.com/community/tutorials/an-introduction-to-oauth-2 
- https://api.slack.com/authentication/oauth-v2#obtaining
- https://docs.tooljet.com/docs/how-to/oauth2-authorization/
- https://auth0.com/docs/get-started/authentication-and-authorization-flow/call-your-api-using-the-authorization-code-flow